app/assets/javascripts/order.js

```diff
@@ -4,11 +4,15 @@ $(document).ready(function() {
   $("#new_order").submit(function(event) {
     var $form = $(this);
     $form.find("button").prop("disabled", true);
-    Stripe.card.createToken($form, stripeResponseHandler);
-    return false;
-  });
-});
-
+    if $("card_number").length;
+      Stripe.card.createToken($form, stripeResponseHandler);
```

Comment:
Expected '{' and instead saw 'Stripe'.<br>'Stripe' is not defined.<br>'stripeResponseHandler' is not defined.